addons/ondevice-knobs/src/panel.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { View, Text, TouchableOpacity } from 'react-native';
 import PropTypes from 'prop-types';
-
+import { SELECT_STORY, FORCE_RE_RENDER } from '@storybook/core-events';
+import { SET, SET_OPTIONS, RESET, CHANGE, CLICK } from '@storybook/addon-knobs/dist/shared';
```

Comment:
We shouldn't be importing from `dist` -- let's export this in `@storybook/addon-knobs/events`?